### ISSUE TITLE
change language

### ISSUE THREAD

**Issue by eddilara** [yaser01/mkv-muxing-batch-gui#27]
would be nice to be able to change the language of a subtitle/audio in the mkv.. without having to demux the audio/sub...and remux them with the proper language

**Comment by yaser01** [comment]
I think this one is heavy because of multiple subtitle/audio in single mkv file 
Can't promise to have it soon but will keep it in mind 
Thanks again ;)

**Comment by yaser01** [comment]
I am sorry but this have to wait a little time :)

**Comment by eddilara** [comment]
basically it would have to read the language tag from the audio video and sub streams...and then have some gui option to tell it to set the the proper language on the tracks
maybe something in the output tab...where you can tell it what language you want to set for each track

I think you really only need 1 changeable language for the video...since its EXTREMELY uncommon for a mkv for have multiple video tracks
for audio and subs...atleast 3-4 changeable languages

**Comment by janaxhell** [comment]
I agree with this request. Being Italian, I use this to mux italian audio, but the only language options in the GUI are English and Arabic. So If I add an Italian audio track to an English video, it will produce a file with 2 audio tracks, both tagged ENG.

**Comment by yaser01** [comment]
@janaxhell you can see this issue https://github.com/yaser01/mkv-muxing-batch-gui/issues/46

**Comment by janaxhell** [comment]
Ouch, sorry! Didn't notice. Honestly it's a little nested, but, thanks.

**Comment by yaser01** [comment]
@janaxhell To be Honest you are right, i will add a way to make it clearer in the next update :)
